oidc-client/Internal.ml

```diff
@@ -21,7 +23,7 @@ let register (type store)
   ( KV.get ~store "dynamic_string" >>= fun dynamic_string ->
     Lwt.return
       (match Oidc.Client.dynamic_of_string dynamic_string with
-      | Error e -> Error e
+      | Error e -> Error (`Msg e)
```

Comment:
Isn't this double-wrapping in `` `Msg ``?

Reply:
Nvm, the signature was changed further down, this is why you don't use "Add single comment"